From 578ef46d7cb57934d51683f1bd373dd2057ee980 Mon Sep 17 00:00:00 2001
From: Tristan <tristan@mangadex.org>
Date: Sun, 9 Jun 2024 13:45:47 +0100
Subject: [PATCH] doc (minor): backport README -> README.md

Without the actual markdown conversion. We don't care and just want the goddamn debian build to pass
---
 .gitignore          | 2 +-
 README => README.md | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename README => README.md (100%)

diff --git a/.gitignore b/.gitignore
index 0e3b22d90..056f84a82 100644
--- a/.gitignore
+++ b/.gitignore
@@ -12,7 +12,7 @@
 !/BRANCHES
 !/BSDmakefile
 !/Makefile
-!/README
+!/README.md
 !/INSTALL
 !/CONTRIBUTING
 !/MAINTAINERS
diff --git a/README b/README.md
similarity index 100%
rename from README
rename to README.md
-- 
2.45.2

